src/js/pages/ReadyLight.jsx

```diff
@@ -66,6 +70,42 @@ class ReadyLight extends Component {
     window.scrollTo(0, 0);
   }
 
+  // Terry - Changes made for WV-1448 adding landing datalayers
```

Comment:
Hi Terry, with a WebStorm feature called Git Blame Annotation, any developer can see who changed any line of code.  So comments like this are very rarely needed, only if there is some complex and non-obvious issue.
Please remove them.